wbooks/app/app.js

```diff
@@ -0,0 +1,105 @@
+(function(){
+  const app = angular.module('store',[]);
+
+  app.controller('StoreController', function(){
+    const store = this;    
+    store.products = books;
+        
+    store.searchBook = function() {
+      store.products = books;
+      if(store.searchBox) {
+        if(store.filterValue === 'title') {
+          store.products = store.products.filter(book => book.title.toLowerCase().indexOf(store.searchBox.toLowerCase()) > -1);  
```

Comment:
Mmmm, this line is being repeated, we can do something about it with the bracket notation to get the attribute of books. For example: `book[store.filterValue]` . That way we can get the value without the need for the if, don't you think? :smile: